### DISCUSSION TITLE
GPU temperature appears then disappears regularly (?)

### DISCUSSION THREAD

**Discussion by Kikimahe** [seerge/g-helper#1286] [General]
Hi

First, thank you for this wonderful software. 🥇

Concerning the GPU temperature in G-Helper, is it normal that it appears then disappears regularly ??

(No display problem for CPU temperature)

Have a nice day. :-)

**Comment by seerge** [comment]
@Kikimahe it's normal when you disable dGPU (i.e. Eco mode or Optimized on battery)

**Comment by Kikimahe** [comment]
Thanks for your reply Seerge. 👍 

I found this strange because in the windows task manager, it is displayed constantly, regardless of the mode used in G-Helper. ;-)

Have a nice day. :-)
